tests/system/feature_store/test_feature_store.py

```diff
@@ -2489,8 +2490,11 @@ def test_online_impute(self):
 
         # create vector and online service with imputing policy
         vector = fs.FeatureVector("vectori", features)
+        vector.save()
```

Comment:
saving the vector was needed to repo the issue?

Reply:
It was needed to use a fvec uri, since it will go to the MLRun DB to retrieve the fvec, and if you didn't save the fvec then you cannot retrieve it...